crypto_secretstream/src/stream.rs

```diff
@@ -30,6 +30,9 @@ use crate::{header::Header, Key, Tag};
 const MAC_BLOCK_SIZE: usize = 16;
 const TAG_BLOCK_SIZE: usize = 16 * 4;
 
+/// Extra bytes per message compared to the plaintext
+pub const ABYTES: usize = MAC_BLOCK_SIZE + 1;
```

Comment:
This isn't visible because the `stream` module is private. So you can:

1. Define it in lib.rs instead
2. Re-export it from lib.rs
3. Make it an inherent constant instead of a freestanding one, i.e. `Stream::ABYTES`

Reply:
I like option 3 the most, so I've done that.